chrome/app/record_replay_main.cc

```diff
@@ -266,9 +266,6 @@ static bool RecordReplayRecordingDisabled(bool cmdline_for_recording) {
       BusyWait();
     return true;
   }
-  if (getenv("RECORD_ALL_CONTENT")) {
-    return false;
-  }
```

Comment:
we shouldn't need this anymore in the renderer processes, since we should always be sending the command line arg from the browser, if `RECORD_ALL_CONTENT` is set in the browser process.